backend/views/syson-diagram-common-view/src/test/java/org/eclipse/syson/diagram/common/view/services/ViewLabelServiceTest.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.diagram.common.view.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.sirius.components.core.api.IFeedbackMessageService;
+import org.eclipse.sirius.components.representations.Message;
+import org.eclipse.syson.sysml.AttributeUsage;
+import org.eclipse.syson.sysml.SysmlFactory;
+import org.eclipse.syson.sysml.helper.LabelConstants;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * View Label services tests.
+ *
+ * @author jmallet
+ */
+public class ViewLabelServiceTest {
+
+    private static final String ATTRIBUTE_USAGE_NAME = "myAttributeUsage";
+
+    private ViewLabelService viewLabelService;
+
+    @BeforeEach
+    void beforeEach() {
+        this.viewLabelService = new ViewLabelService(new TestFeedbackMessageService(), new ShowDiagramsIconsService());
```

Comment:
Use `new IFeedbackMessageService.NoOp()` instead of `TestFeedbackMessageService()`

Reply:
Done